### ISSUE TITLE
(node:18220) UnhandledPromiseRejectionWarning: TypeError: Cannot read property ' error' of undefined

### ISSUE THREAD

**Issue by element4l** [mmgoodnow/cross-seed#117]
Really not sure what to do here. I'm on windows 7 and installed cross-seed globally

```
C:\Users\...\Downloads\Programs\cross-seed-2.5.0\cross-seed-2.5.0>cross-seed s
earch
(node:18220) UnhandledPromiseRejectionWarning: TypeError: Cannot read property '
error' of undefined
    at CrossSeedError.print (C:\Users\...\AppData\Roaming\npm\node_modules\cro
ss-seed\dist\src\errors.js:11:25)
    at Command.<anonymous> (C:\Users\...\AppData\Roaming\npm\node_modules\cros
s-seed\dist\src\cmd.js:114:19)
    at Command.listener [as _actionHandler] (C:\Users\...\AppData\Roaming\npm\
node_modules\cross-seed\node_modules\commander\index.js:922:31)
    at Command._parseCommand (C:\Users\...\AppData\Roaming\npm\node_modules\cr
oss-seed\node_modules\commander\index.js:1503:14)
    at Command._dispatchSubcommand (C:\Users\...\AppData\Roaming\npm\node_modu
les\cross-seed\node_modules\commander\index.js:1443:18)
    at Command._parseCommand (C:\Users\...\AppData\Roaming\npm\node_modules\cr
oss-seed\node_modules\commander\index.js:1460:12)
    at Command.parse (C:\Users\Slice\AppData\Roaming\npm\node_modules\cross-seed
\node_modules\commander\index.js:1292:10)
    at Command.parseAsync (C:\Users\...\AppData\Roaming\npm\node_modules\cross
-seed\node_modules\commander\index.js:1318:10)
    at run (C:\Users\...\AppData\Roaming\npm\node_modules\cross-seed\dist\src\
cmd.js:121:31)
    at Object.<anonymous> (C:\Users\...\AppData\Roaming\npm\node_modules\cross
-seed\dist\src\cmd.js:123:1)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:18220) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This
 error originated either by throwing inside of an async function without a catch
 block, or by rejecting a promise which was not handled with .catch(). To termin
ate the node process on unhandled promise rejection, use the CLI flag `--unhandl
ed-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejecti
ons_mode). (rejection id: 2)
(node:18220) [DEP0018] DeprecationWarning: Unhandled promise rejections are depr
ecated. In the future, promise rejections that are not handled will terminate th
e Node.js process with a non-zero exit code.
```

**Comment by mmgoodnow** [comment]
What's your config file and command line arts?

**Comment by element4l** [comment]
I have now fixed it. In case anyone is running into the same issue, try making sure your paths use forward slashes instead of backward slashes. I wish I could say what else went wrong, but it was a bit of quick fidgeting and troubleshooting over a couple of drinks. Appreciate the note to have a closer look at the config file. All up and running now.

**Comment by mmgoodnow** [comment]
Which paths?

**Comment by element4l** [comment]
For example, C:/Users/.../AppData/Local/qBittorrent/BT_backup

**Comment by mmgoodnow** [comment]
I looked into this and fixed a bug that was causing this crash in v2.5.1. It threw an exception while trying to print you an error, which specifically was `"You need to specify --rtorrent-rpc-url or --qbittorrent-url when using '-A inject'."` Windows-style paths should be fine to use.
